README.md

```diff
@@ -65,25 +65,20 @@ sticky sessions, refer to the [Session Affinity document](docs/session-affinity.
 [X-CF Headers](/docs/x_cf_headers.md) describes the X-CF headers that are set on requests and responses inside of CF.
 
 ## <a name="routing-contributor-resources"></a> Routing Contributor Resources
-### <a name="ci-statues"></a> CI Statuses
-Job | Status
---- | ---
-unit tests | [![networking.ci.cf-app.com](https://networking.ci.cf-app.com/api/v1/teams/ga/pipelines/routing/jobs/routing-release-unit/badge)](https://networking.ci.cf-app.com/teams/ga/pipelines/routing/jobs/routing-release-unit)
-performance tests | [![networking.ci.cf-app.com](https://networking.ci.cf-app.com/api/v1/teams/ga/pipelines/routing/jobs/diana-tcp-perf-tests/badge)](https://networking.ci.cf-app.com/teams/ga/pipelines/routing/jobs/diana-tcp-perf-tests)
-smoke tests | [![networking.ci.cf-app.com](https://networking.ci.cf-app.com/api/v1/teams/ga/pipelines/routing/jobs/cf-deployment-smoke-and-indicator-protocol-tests/badge)](https://networking.ci.cf-app.com/teams/ga/pipelines/routing/jobs/cf-deployment-smoke-and-indicator-protocol-tests)
 
 ### <a name="developer-workflow"></a> Developer Workflow
 
-1. Clone CI repository (next to where routing-release is cloned)
+- Clone CI repository (next to where this code is cloned)
```

Comment:
Can you please link the CI repository?

Reply:
The link is present in the below snippet, but I've gone ahead and added it as a link too.